src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-               optionsBuilder.UseSqlServer("Data Source=5CD1199VTK;Initial Catalog=PatientCheckIn;Integrated Security=True");
+               optionsBuilder.UseSqlServer("Data Source=5CD1199VDQ;Initial Catalog=PatientCheckIn;Integrated Security=True");
             }
```

Comment:
Connection strings should be retrieved from appsettings.

Reply:
Connnection string has been defined in appsettings file and used in startup